source/gui/settingsDialogs.py

```diff
@@ -4041,17 +4050,6 @@ def __init__(self, parent):
 		self.trimLeadingSilenceCheckBox.SetValue(config.conf["speech"]["trimLeadingSilence"])
 		self.trimLeadingSilenceCheckBox.defaultValue = self._getDefaultValue(["speech", "trimLeadingSilence"])
 
-		# Translators: This is the label for a combo-box control in the
-		#  Advanced settings panel.
-		label = _("Use WASAPI for SAPI 4 audio output:")
-		self.useWASAPIForSAPI4Combo = speechGroup.addLabeledControl(
```

Comment:
I think we need to keep this until 2026.1 for backwards compatibility reasons.

Reply:
Is there anything against re-renabling the link config values feature that is currently commented out? See config/__init__.py line 1307 and below.